packages/neo-one-server-plugin-wallet/src/bootstrap.js

```diff
@@ -39,16 +39,35 @@ import constants from './constants';
 import { kycContract, conciergeContract } from './__data__/contracts';
 
 const DEFAULT_NUM_WALLETS = 10;
-const NEOTRACKERRPC = 'http://localhost:1351/rpc';
-const NEOTRACKERMASTERPRIVATEKEY =
+const NEOTRACKER_MASTER_PRIVATE_KEY =
   '9e9522c90f4b33cac8a174353ae54651770f3f4dd1de78e74d9b49ba615d7c1f';
-const NEOTRACKERNETWORKNAME = 'priv';
-const NEOTRACKERPRIVATEKEYS = [
+const NEOTRACKER_NETWORK_NAME = 'priv';
```

Comment:
Probably shouldn't be called `NEOTRACKER` anymore since it's not specific to neotracker.